tools/system_libs.py

```diff
@@ -2059,7 +2062,7 @@ class libsanitizer_common_rt(CompilerRTLibrary, MTLibrary):
               'system/lib/compiler-rt/lib',
               'system/lib/libc']
   never_force = True
```

Comment:
Add a comment/TODO here with link the upstream issue?

Reply:
Not sure if there is a specific upstream issue for this. Will add a todo saying we can remove this after upstream format problems are resolved.

Reply:
Will paste a link to https://github.com/llvm/llvm-project/blob/da675b922cca3dc9a76642d792e882979a3d8c82/compiler-rt/lib/sanitizer_common/CMakeLists.txt#L225-L226 then.